easybuild/easyconfigs/t/Theano/Theano-1.0.0-intel-2017b-Python-2.7.14.eb

```diff
@@ -16,6 +16,12 @@ checksums = ['7bc82e4fcaac023cac23c390b30d5df81bc09d33568812df75b0bd2b86d2fa8f']
 
 dependencies = [('Python', '2.7.14')]
 
+# Save the compiledir on the local disk to avoid issues,
+# when running the software on a heterogeneous compute cluster.
+# Change temporary directory "/tmp" and  environment variable "USER" to match your cluster setup.
+import os
```

Comment:
@smoors Hmm, not too happy with the `import os` here...

Also, it doesn't make sense, since this will make `Theano` use the the `$USER` value of the installation user, rather than the value of `$USER` that is *using* Theano.